src/main/java/exceptions/DeadlineDateError.java

```diff
@@ -0,0 +1,4 @@
+package exceptions;
+
+public class DeadlineDateError extends RuntimeException {
```

Comment:
Inheriting from RuntimeException is not good in this scenario. You should use subclasses of Exception class so that you can handle the exceptions in your codebase. 
Additionally, better to have individual exception handling (or atleast specific error messages) for each type of exception instead of having empty classes